addons/binding/org.openhab.binding.enocean/README.md

```diff
@@ -0,0 +1,128 @@
+
+### Sensors
+
+To pair a sensor with its thing, you first have to start the discovery scan for this binding in PaperUI. Then press the "teach-in" button of the sensor. The sensor sends a teach-in message which contains the information about the EEP and the device Id of the sensor. If the EEP is known by this binding the thing representation of the device is created. The corresponding channels are created dynamically, too. 
+
+### Actuators
+ 
+If the actuator supports UTE teach-ins, the corresponding thing can be created and paired automatically. First you have to start the discovery scan for this binding in PaperUI. Then press the teach-in button of the actuator. 
+
+If the actuator does not support UTE teach-ins, you have to create, configure and choose the right EEP of the thing manually. It is important to link the teach-in channel of this thing. Afterwards you have to activate the pairing mode of the actuator. Then switch on the teach-in item(/channel) to send a teach-in message to the actuator. If the pairing was successful, you can control the actuator and unlink the teach-in channel.
+
+
+## Adding Enocean-Things (Actuators, Switches, Sensors, etc.)
+
+https://github.com/fruggy83/enocean/wiki#how-to-add-an-enocean-thing-actuator-sensor-wallswitch-etc-via-paperui
+
+## Thing Configuration
+
+Things can and should by configured through PaperUI. Following the most important config parameters:
+
+Bridge
+
+ * Serial port: The serial port to which the EnOcean gateway is connected to
+ * Next device id: The device Id which should be taken for the next created actuator without an explicit device Id
+ 
+Things
+
+ * Thing Id: EnOcean Id of the device. 4 byte hex string (abcdef00).
+ * Bridge: Must be provided
+ * Sender Id: Is used to generate the unique EnOcean device Id for sending messages (added to the base Id of the gateway). If you leave it empty, the next free Id will be determined automatically (see "next device id" of bridge). The resulting device Id can be seen through the properties.
+ * (Sending/receiving) EEP: Set the EEP which should be used to send and receive message to or from the device.
+ 
+## Generic Things
+
+  If an Enocean device uses an unsupported EEP or _A5-3F-7F_, you have to create a generice thing. Generic things support all generic channels like switch, number, string etc. However you have to specify how to convert the Enocean messages from the thing into Openhab state updates and how to convert the Openhab commands into Enocean messages. These conversion functions can be defined with the help of transformation functions like MAP or JavaScript.
+
+For an inbound transformation (Enocean message => Openhab state) you get the channel id and the Enocean data seperated by a pipe.
+
+```
+ChannelId|EnoceanData(Hex)=OpenhabState|Value
```

Comment:
This isn't clear to me.  Where is this information entered?

Reply:
I am using the transform functionality of openHAB in this case. So you have to enter these information in files and put them into the transform folder of openHAB. Updated the doc regarding this point.